pkgs/development/interpreters/ueforth/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchFromGitHub
+, nodejs, python
+}:
+let
+  REVISION = "830e6b40d7d3a4c67302c65359c4989e2ff5a5de";
+in stdenv.mkDerivation {
+  name = "ueforth";
+  inherit REVISION;
```

Comment:
Do we need this as an env at all? this is already handed to make below.

Reply:
fixed